pkg/solana/client/multinode/poller.go

```diff
@@ -65,7 +65,8 @@ func (p *Poller[T]) Err() <-chan error {
 }
 
 func (p *Poller[T]) pollingLoop(ctx context.Context) {
-	ticker := time.NewTicker(p.pollingInterval)
+	tickerCfg := services.TickerConfig{Initial: 0, JitterPct: services.DefaultJitter}
+	ticker := tickerCfg.NewTicker(p.pollingInterval)
```

Comment:
isn't this redundant with the default? [`services.NewTicker`](https://github.com/smartcontractkit/chainlink-common/blob/ad84e3712352552a7c8720860aed7942d01228c8/pkg/services/ticker.go#L16)